test/e2e/multihoming.go

```diff
@@ -797,6 +800,36 @@ var _ = Describe("Multi Homing", func() {
 							return inRange(newDesiredRange, clientIP)
 						}).Should(Succeed(), "pod's secondary NIC is not allocated in the desired range")
 					})
+
+					It("can no longer communicate over a localnet secondary network from pod to the underlay service", func() {
+						clientPodConfig := podConfiguration{
+							name:        clientPodName,
+							namespace:   f.Namespace.Name,
+							attachments: []nadapi.NetworkSelectionElement{{Name: secondaryNetworkName}},
+						}
+						kickstartPod(cs, clientPodConfig)
+
+						By("asserting the *client* pod can no longer contact the underlay service")
+						Eventually(connectToServer).WithArguments(clientPodConfig, underlayServiceIP, servicePort).ShouldNot(Succeed())
```

Comment:
can we match the error here ? something like: https://github.com/ovn-kubernetes/ovn-kubernetes/blob/96a9350ccab1ceb69029b84bcec145945c7ffc6f/test/e2e/multihoming.go#L887

Reply:
[DONE](https://github.com/ovn-kubernetes/ovn-kubernetes/compare/96a9350ccab1ceb69029b84bcec145945c7ffc6f..24b2bb455229a5f8a223f8b20ce175cd681a40fa)